test/remote/check_tests.js

```diff
@@ -88,6 +98,49 @@ test(
   }
 )
 
+test('allowed email addresses in /check do not block subsequent requests to /checkIpOnly', t => {
```

Comment:
This test mimics what was happening in the functional tests when we noticed the problem.